### PR TITLE
Repo cleanup

### PR DESCRIPTION
**Pull request by eric-schuh**
General respec and terms cleanup, some spelling/grammar updates

***
<a href="https://pr-preview.s3.amazonaws.com/eric-schuh/vc-api-use-cases/pull/29.html" title="Last updated on Mar 14, 2023, 6:44 PM UTC (93c35c6)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/vc-api-use-cases/29/dac148c...eric-schuh:93c35c6.html" title="Last updated on Mar 14, 2023, 6:44 PM UTC (93c35c6)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -331,7 +309,7 @@ <h2>User Roles</h2>
   <section>
     <h2>User Needs</h2>
       <p>
-        The <a>VC-API</a> address user needs in a number of key domains:
+        The VC-API address user needs in a number of key domains:
```
Comment (TallTed) [inline index.html:312]:
```suggestion
        The VC-API addresses user needs in a number of key domains:
```

File: terms.html [hunk 2]
```diff
@@ -167,4 +167,27 @@
   <dd>
 A Uniform Resource Identifier, as defined by [[RFC3986]].
   </dd>
-</dl>
\ No newline at end of file
+  <dt><dfn data-lt="Issuer Service"></dfn>issuer service</dt>
+  <dd>
+        The <a>component</a> that creates, generates, and delivers a <a>Verifiable Credential</a> — which contains
+        a some set of <a>claims</a> made by the Issuer Coordinator about one or more subjects — to the Issuer Coordinator.
```
Comment (TallTed) [inline terms.html:173]:
```suggestion
        some set of <a>claims</a> made by the Issuer Coordinator about one or more subjects — to the Issuer Coordinator. 
```

File: terms.html [hunk 3]
```diff
@@ -167,4 +167,27 @@
...
+        The <a>component</a> performing the verification of a <a>Verifiable Credential</a> containing <a>claims</a> about a given <a>subject</a>.
+  </dd>
+    <dt><dfn data-lt="Verifier Coordinator"></dfn>verifier coordinator</dt>
+  <dd>
+        The <a>component</a> that deals with business rules and policies around a
+        user's request for a particular <a>Verifiable Credential</a> to be verified.
+  </dd>
+    <dt><dfn data-lt="component"></dfn>component</dt>
+  <dd>
+    A representation of a set of tasks under the perview of a single <a>entity</a> which are indented to be implimentable seperately from other components.
```
Comment (TallTed) [inline terms.html:191]:
```suggestion
    A representation of a set of tasks under the purview of a single <a>entity</a> which are indented to be implementable separately from other components. 
```

**Review by TallTed — CHANGES_REQUESTED** [review]
A few minor fixes to the fixes

**Comment by jandrieu** [comment]
Looks good. 

I think we'll want to make some policy choices about capitalization. I think Issuer, Verifier, and Holder are legitimate proper nouns, referring, e.g., to *the* Holder in a given interaction.

My thought is that when referring to the roles, that makes sense and is a bit of a different usage than my lazy overcapitalization of things like "method" in "DID method", which should not be capitalized (even though my fingers keep typing it that way.)

I'm sure there are other places to pay attention to that, but I'm going to merge. And add an issue.

**Pull request merged**